addon/components/dropzone-widget/component.js

```diff
@@ -24,34 +24,42 @@ export default Ember.Component.extend({
     layout,
     session: Ember.inject.service(),
     classNames: ['dropzone'],
-    didRender() {
-        var preUpload = this.get('preUpload');
-        var dropzoneOptions = this.get('options');
-        var listeners = this.get('listeners');
+    didInsertElement() {
+        let preUpload = this.get('preUpload'),
```

Comment:
Please break this into two statements (and see https://github.com/CenterForOpenScience/ember-osf/blob/develop/.github/CONTRIBUTING.md#checking-your-code) 

Your travis builds will fail until `npm run check-style` is clean